playground/mocks/data/idp/idx/challenge-with-push-notification.json

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
can we rename this file to be similar to the other `challenge-poll` remediation files? Something like `authenticator-verification-okta-verify-push-autoChallenge.json`?
And maybe some suffix like that for the other files that are already similar to the files we currently have. It seems that the only difference is the `autoChallenge` bit, but the rest of the forms are the same